web_responsive/static/src/less/form_view.less

```diff
@@ -57,13 +57,16 @@
     >div {
         >.table-responsive {
             >.o_list_view {
-                thead {
+                thead tr:nth-child(1) th {
                     position: sticky;
                     top: 0;
+                    background-color: #EEEEEE;
```

Comment:
isn't there any variable for this color?

Reply:
What do you mean ?

Reply:
`@odoo-color-for-thead` -> likely there's a variable to reuse and you don't hardcode this color

Reply:
https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/less/list_view_extra.less#L4 :man_shrugging:

Reply:
amazing :rofl: